content/docs/reference-react-dom-server.md

```diff
@@ -17,109 +17,113 @@ var ReactDOMServer = require('react-dom/server');
 
```

Comment:
The ESM import is incorrect here. It should be `import * as `.

We shouldn't recommend `import *` neither since it breaks DCE but I guess we do that every where in old docs. It matters a bit for the server since we pull in multiple copies of the entire bundles through this one entry point.